pkg/crc/api/handlers.go

```diff
@@ -54,6 +54,7 @@ func (h *Handler) Status(c *context) error {
 		CrcStatus:        string(res.CrcStatus),
 		OpenshiftStatus:  string(res.OpenshiftStatus),
 		OpenshiftVersion: res.OpenshiftVersion,
+		PodmanVersion:    res.PodmanVersion,
```

Comment:
This adds too much information and makes it hard to determine the type:

![image](https://user-images.githubusercontent.com/1894/146758353-3d026829-cfba-40cb-a85f-8da2983b4e53.png)

can you either add `preset` or only add the necessary fields.

Reply:
Added in another PR